platform/linuxbsd/display_server_x11.cpp

```diff
@@ -1943,28 +1951,30 @@ String DisplayServerX11::keyboard_get_layout_name(int p_index) const {
 }
 
 DisplayServerX11::Property DisplayServerX11::_read_property(Display *p_display, Window p_window, Atom p_property) {
-	Atom actual_type;
-	int actual_format;
-	unsigned long nitems;
-	unsigned long bytes_after;
+	Atom actual_type = None;
+	int actual_format = 0;
+	unsigned long nitems = 0;
+	unsigned long bytes_after = 0;
 	unsigned char *ret = nullptr;
 
 	int read_bytes = 1024;
 
 	//Keep trying to read the property until there are no
-	//bytes unread.
-	do {
-		if (ret != nullptr) {
-			XFree(ret);
-		}
+	//bytes unread
```

Comment:
I guess the dot was not removed on purpose, but might as well fix the comment style:
```
// Keep trying to read the property until there are no bytes unread.
```